cmd/main.go

```diff
@@ -87,9 +103,20 @@ type config struct {
 	serverCert     string
 	serverKey      string
 
+	httpConfig HTTPConfig
```

Comment:
```suggestion
	httpHost	string
	httpPort	string
	httpTargetHost	string
```
To be the same as other configs